src/app/containers/RecentAudioEpisodes/index.jsx

```diff
@@ -116,12 +119,20 @@ const RecentAudioEpisodes = ({ episodes }) => {
             </EpisodeList.Link>
             {episode.episodeTitle && (
               <EpisodeList.Metadata>
-                {' '}
-                <StyledSpan aria-hidden>|</StyledSpan>
-                {formattedTimestamp({
-                  releaseDateTimeStamp: episode.timestamp,
-                  format: 'LL',
-                })}
+                <>
+                  {' '}
+                  <StyledSpan aria-hidden>|</StyledSpan>
```

Comment:
Have we removed this elsewhere?

Reply:
yep @pharingee fixed this https://github.com/bbc/psammead/pull/4125